diplom.py

```diff
@@ -9,8 +9,27 @@ def parse(line):
     label = int (tokens[23])
     return (features, label)
 
+def mean_average_precision (predicted_probability_distribution, 
+    n_of_classes, 
+    list_of_classes, 
+    test_y, 
+    dataset_size):
+    AP = 0
+    k = -1
+    for row in predicted_probability_distribution:
+        k += 1
+        classes_by_prob = []
+        for i in xrange(0, n_of_classes):
+            classes_by_prob.append([row[i], list_of_classes[i]])
+            classes_by_prob.sort(reverse=True)    
+        for j in range (0, 5):
+            if classes_by_prob[j][1] == test_y[k]:
+                AP += 1.0 / (j+1)
+    MAP = AP / dataset_size
+    return MAP
+
 TRAIN_DATASET_SIZE = 30000
-TEST_DATASET_SIZE = 30000
+dataset_size = 30000
```

Comment:
Rename this constant back to TEST_DATASET_SIZE. I've asked you to only rename argument of the function ;)